packet/packet.py

```diff
@@ -0,0 +1,38 @@
+from .models import *
+
+
+def sign(member_username, freshman_username):
+    freshman = Freshman.query.filter_by(rit_username=freshman_username)[0]
+    packet = freshman.current_packet()
+    if packet is None:
+        return False
+    if not packet.is_open():
+        return False
+    if UpperSignature.query.filter_by(member=member_username, eboard=True)[0]:
```

Comment:
Why are there separate queries here for the eboard/non-eboard cases? Wouldn't a single query which ignores `eboard` suffice?

Reply:
Depending on devin's desire for implementation this can be revised.  The idea is that eboard will show up first, or have their own visible categorization to indicate them as eboard.  Doing it this way means devin doesn't have to iterate a list twice to grab eboard and non-eboard

Reply:
Displaying eboard separately in the UI is fine, but this is just the backend implementation of the signing API call - it doesn't affect how anything will be rendered.

Reply:
As long as the attribute `EBoard` is available in the objects returned, I'll handle it all in the frontend